shader/forward.frag

```diff
@@ -212,4 +212,5 @@ void main()
     write_gbuffer_pos(v.pos);
     write_gbuffer_screen_motion(get_camera_projection(camera.pairs[gl_ViewIndex].previous, v.prev_pos));
     write_gbuffer_instance_id(int(control.instance_id));
+    write_gbuffer_linear_depth();
```

Comment:
Linear depth as a separate output is a bit weird as you should be able to just use the depth buffer instead, and linearize it. But I see that it can be more convenient and is probably not a bottleneck for now (esp. since it's only used in SVGF).